### ISSUE TITLE
error in Juno

### ISSUE THREAD

**Issue by AStupidBear** [JuliaPlots/PlotRecipes.jl#16]
![2017-03-23_100333](https://cloud.githubusercontent.com/assets/16422976/24228571/167a544a-0fb0-11e7-9067-34acbc849ac5.jpg)

**Comment by mkborregaard** [comment]
Yes I get this  all the time from StatPlots and PlotRecipes (that imports StatPlots). However, none of those packages depend on Media - but Juno and Atom do. Maybe try to open an issue on Juno and link this?

**Comment by mkborregaard** [comment]
cf https://github.com/JunoLab/atom-julia-client/issues/323
